ethcore/src/types/trace_types/filter.rs

```diff
@@ -71,7 +71,7 @@ impl AddressesFilter {
 				.flat_map(|bloom| self.list.iter()
 					.map(|address| bloom.with_bloomed(&address.sha3()))
```

Comment:
future work: pre-hashing the addresses would probably save a fair bit of time here